Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -389,6 +389,11 @@ void UpdateCharacterSpacing()
 
 			if (Element?.TextType != TextType.Text)
 				return;
+
+			if (Element.CharacterSpacing == 0)
```

Comment:
I think this will make it so you can't reset CharacterSpacing to 0.

Reply:
Removed this unnecessary check. Probably left after testing.